R/pkg/R/DataFrame.R

```diff
@@ -570,10 +570,17 @@ setMethod("unpersist",
 
 #' Repartition
 #'
-#' Return a new SparkDataFrame that has exactly numPartitions partitions.
+#' There are two different options for repartition
+#' Option 1
```

Comment:
roxygen2 by default strip out all the whitespace and new line.
if you want you could put these into \items or trail by \cr
http://stackoverflow.com/questions/9267584/when-documenting-in-roxygen-how-do-i-make-an-itemized-list-in-details
http://r-pkgs.had.co.nz/man.html